appsmithorg/appsmith
# fix: changed the default row size to 4 for the divider widget

## Description

I have increased the default row size of the divider widget to 4. This will help the users to expand the divider widget in a convenient manner.

Fixes #9642 

## Type of change

- Bug fix (non-breaking change which fixes an issue)

## How Has This Been Tested?

This has been tested manually. 

Steps to test:

1. Drag and drop the divider widget on to the canvas.
2. Change the height along the y-axis of the widget.
3. Observe.

[![LOOM DEMO](http://cdn.loom.com/sessions/thumbnails/51ea40c598cf419a933112372f1813ca-00001.gif)](https://www.loom.com/share/51ea40c598cf419a933112372f1813ca)

## Checklist:



## Test coverage results :test_tube:
<details><summary>:green_circle: Total coverage has increased</summary>


    // Code coverage diff between base branch:release and head branch: fix/fix-divider_height_expand 
Status | File | % Stmts | % Branch | % Funcs | % Lines 
 -----|-----|---------|----------|---------|------ 
 :green_circle: | total | 55.22 **(0)** | 37.02 **(0.01)** | 34.07 **(0)** | 55.73 **(0)**
 :green_circle: | app/client/src/utils/autocomplete/TernServer.ts | 52.94 **(0.23)** | 41.67 **(0.84)** | 36.21 **(0)** | 56.99 **(0.25)**</details>
